src/lightgbm/src/main/scala/TrainUtils.scala

```diff
@@ -35,8 +36,27 @@ private object TrainUtils extends Serializable {
     }
   }
 
+  private def addIntField(field: Array[Int], fieldName: String, numRows: Int,
```

Comment:
Does this need to be here with the PR that just got merged? Hoping we can remove some of the SWIGTYPE_p_blahs now

Reply:
done!  moved to datasets file when rebasing to latest